storage/innobase/handler/ha_innodb.cc

```diff
@@ -683,8 +687,11 @@ ha_create_table_option innodb_table_option_list[]=
   HA_TOPTION_NUMBER("PAGE_COMPRESSION_LEVEL", page_compression_level, 0, 1, 9, 1),
   /* With this option the user can enable encryption for the table */
   HA_TOPTION_ENUM("ENCRYPTED", encryption, "DEFAULT,YES,NO", 0),
-  /* With this option the user defines the key identifier using for the encryption */
+  /* With this option the user defines the key identifier used for the encryption */
   HA_TOPTION_SYSVAR("ENCRYPTION_KEY_ID", encryption_key_id, default_encryption_key_id),
```

Comment:
How about "This option identifies the key for encrypting the data file"? This would be best changed in a comment on its own, though.

Reply:
Correct. This change is unrelated. I'll revert in a new push.